### PR TITLE
flipcard/bottom nav issue

### PR DESCRIPTION
**Pull request by warwickandrew**
fixes jumping movement when flipcard and footer nav is used together

### CODE REVIEW

File: less/flipcard.less [hunk 1]
```diff
@@ -30,21 +30,27 @@
...
-
     .flipcard-item-face {
       width: 100%;
     }
   }
+  
+   .flipcard-single {
+     width: 100%;
+
+      flipcard-item-face {
```
Comment (nilslp) [inline less/flipcard.less]: Can you double check that this is actually correct.
Looks like the `.` for the class is missing here.

**Review by nilslp — CHANGES_REQUESTED** [review]
Could you take a look at the comment I left please.
Then bump the version of the plugin, Thanks

File: bower.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
     "name": "adapt-contrib-flipcard",
     "framework": ">=3.0.0",
-    "version": "3.0.0",
-    "homepage": "https://github.com/learningpool/adapt-contrib-flipcard",
-    "issues": "https://github.com/learningpool/adapt-contrib-flipcard/issues",
-    "repository": "git://github.com/learningpool/adapt-contrib-flipcard.git",
+    "version": "3.0.1",
+    "homepage": "https://github.com/ExultCorp/adapt-contrib-flipcard",
+    "issues": "https://github.com/ExultCorp/adapt-contrib-flipcard/issues",
+    "repository": "git://github.com/ExultCorp/adapt-contrib-flipcard.git",
```
Comment (cahirodoherty-learningpool) [inline bower.json]: These changes should not have been reintroduced

File: less/flipcard.less [hunk 3]
```diff
@@ -30,21 +30,27 @@
...
-    .flipcard-item-face {
-      width: 99%;
-    }
-  }
-
-  .flipcard-single {
-    width: 100%;
-
     .flipcard-item-face {
       width: 100%;
```
Comment (cahirodoherty-learningpool) [inline less/flipcard.less]: Should this be 99% I think a merge conflict resolution has made this creep in

**Review by cahirodoherty-learningpool — CHANGES_REQUESTED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]

**Review by danielstorey — APPROVED** [review]

**Review by nilslp — APPROVED** [review]

**Pull request merged**